lib/model/maalcie/MaaltijdAanmeldingenModel.class.php

```diff
@@ -203,53 +212,22 @@ public static function getAanmeldingenVoorLid($maaltijdenById, $uid) {
 		return $result;
 	}
 
-	public static function getIsAangemeld($mid, $uid, $doorAbo = null) {
+	public function getIsAangemeld($mid, $uid) {
         $aanmelding = new MaaltijdAanmelding();
         $aanmelding->maaltijd_id = $mid;
         $aanmelding->uid = $uid;
-        if ($doorAbo) {
-            $aanmelding->door_abonnement = $doorAbo;
```

Comment:
Dit is voor het weergeven van "(abo)" bij een aanmelding voor de gebruiker denk ik.